lib/charms/mongodb/v1/mongodb_provider.py

```diff
@@ -14,8 +14,8 @@
 from typing import List, Optional, Set
 
 from charms.data_platform_libs.v0.data_interfaces import DatabaseProvides
-from charms.mongodb.v0.mongodb import MongoDBConfiguration, MongoDBConnection
 from charms.mongodb.v1.helpers import generate_password
+from charms.mongodb.v1.mongodb import MongoDBConfiguration, MongoDBConnection
```

Comment:
this library will need to be bumped to libpatch 7 in order for your change to be published [it is currently at libpatch 6 on charmhub](https://charmhub.io/mongodb/libraries/mongodb_provider) and your change will probably result in some failing charmlib checks